packages/genesys-spark-components/stencil.config.ts

```diff
@@ -9,6 +10,16 @@ const testConsoleReporter =
     ? 'default'
     : ['jest-silent-reporter', { useDots: true }];
 
+// Optionally host the dev server with https if a cert and key are provided via env variables, e.g.
+// for running local dev build within a https app using assetsUrl option of registerSparkComponents.
+let https: Credentials | undefined = undefined;
+if (process.env.SPARK_HTTPS_CERT && process.env.SPARK_HTTPS_KEY) {
+  https = {
+    key: process.env.SPARK_HTTPS_KEY,
+    cert: process.env.SPARK_HTTPS_CERT
+  };
+}
+
```

Comment:
This part is not strictly necessary, but it goes hand-in-hand. If my localhost app uses https, then HSTS will prevent the browser from loading assets from a non-https localhost dev server. My preference would be to add a dependency on @genesys/localcert but that package is in our private repo so the next best thing is giving myself a way to run with localcert sans dependency.

Now I can do this to get a localhost dev server:

```shell
> SPARK_HTTPS_CERT=$(npx -y @genesys/localcert produce certificate -c) SPARK_HTTPS_KEY=$(npx -y @genesys/localcert produce certificate -k) npm run dev.gsc
```